src/WorkflowStep.ts

```diff
@@ -24,23 +24,28 @@ export interface StepConfigureArguments {
 }
 
 export interface StepUpdateArguments {
-  inputs?: {};
-  outputs?: [];
+  inputs?: {
+    [key: string]: {
+      value: any;
+      skip_variable_replacement?: boolean;
+      variables?: {
+        [key: string]: any
+      };
+    }
+  };
+  outputs?: [{
+    name: string;
+    type: string;
+    label: string;
+  }];
   step_name?: string;
   step_image_url?: string;
 }
 
 export interface StepCompleteArguments {
-  inputs?: {
-    [key: string]: {
-      value: string;
-    };
-  };
   outputs?: {
-    type: string;
-    name: string;
-    label: string;
-  }[];
+    [key: string]: any;
```

Comment:
For TypeScript users, a new patch version fixing this should be released as soon as possible.

Reply:
Great catch on this!

Reply:
Should this be `string`, rather than `any`?

Reply:
I'm still not sure it's safe enough to assume the value is always a string. Do you have a source for it?

Reply:
I echo your sentiment. I'm not 100% confident to assume it's always a string at this point.